pandapower/plotting/collections.py

```diff
@@ -995,18 +998,17 @@ def create_load_collection(net, loads=None, size=1., infofunc=None, orientation=
     """
     loads = get_index_array(loads, net.load.index)
     infos = [infofunc(i) for i in range(len(loads))] if infofunc is not None else []
-    node_coords = net.bus_geodata.loc[net.load.loc[loads, "bus"].values, ["x", "y"]].values
```

Comment:
To avoid overlaps of patches, complete situation of a bus is needed to be known for i.e if there are sgens and gens there as well, hence it is replaced by 
node_coords = net.bus_geodata.loc[:, ["x", "y"]].values